src/views/v2/top-bar/nav.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * View: Top Bar - Navigation
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/top-bar/nav.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-calendar-top-bar--nav">
+	<nav>
+		<a href="#" class="tribe-common-c-btn-icon tribe-common-c-btn-icon--caret-left tribe-common-b3"></a>
```

Comment:
All our navs should be a `<ul>` with each link wrapped in `<li>`. So for example:

```
<nav class="">
    <ul class="">
        <li class="">
            <a href="#" class=""></a>
        </li>
        ...
    </ul>
</nav>
```